client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -1820,8 +1821,8 @@
                     });
 
                     superdesk.intent('edit', 'crop',  scope.item).then(function(data) {
-                        if (!scope.$parent.dirty) {
-                            scope.$parent.dirty = data.isDirty;
+                        if (mainEditScope.hasOwnProperty('dirty') && !mainEditScope.dirty) {
```

Comment:
The `hasOwnProperty()` should not be here. If we assume that we are using the correct scope, it will always evaluate to `true`), making it redundant, but if we cannot assume that, this check should result in an error IMO. As it currently stands, it would silently result in a bug (the `dirty` flag will never get set there).